scripts/migration/initialize_menas.py

```diff
@@ -159,6 +159,7 @@ def run(parsed_args: argparse.Namespace):
             print(f"DB '{target_db_name}' contains version record, but it is invalid: {version_err}.\n")
             exit(10)
         else:
+            assert found_version != 1  # this would not be a valid MenasDbVersionError
```

Comment:
This line makes the bellow if a bit obsolete, doesn't it?

Reply:
What? if the version is not 1, it can be lower or higher (in theory). Or are you suggesting to rely on the fact that version=0 should never appear, because the pre-1 Menas DBs did not have `db_version` at all?

Reply:
So, AFAIK, assert will throw an exception if `found_version not equal 1`. On line 163 there is if `found_version lower than 1` (also true if not equal). So the line on 163 is never reached thanks to the assertion on line 162. Right?

Reply:
> AFAIK, assert will throw an exception if `found_version not equal 1`

No, assert will throw an exception if `found_version` **is equal to 1**. So we have a guarantee it is not equal to one and we process further to throw the appropriate error message by comparison to the actual `found_version`.

Reply:
My bad

Reply:
No worries 👍